### ISSUE TITLE
pytest in version >=6.1.0 causes errors

### ISSUE THREAD

**Issue by matteoosh** [ElSnoMan/pyleniumio#126]
![image](https://user-images.githubusercontent.com/6298227/94410882-31509500-0178-11eb-9983-40abad5b3b96.png)
This is caused by newest pytest release (v.6.1.0 on Sep 26).

As a temporary solution I would go with Pipfile entry
`pytest = "<6.1.0"`
until solution is found.

**Comment by matteoosh** [comment]
Runing `pytest --help` returns something like this:
```
warning : /Users/mateuszsiewniak/.local/share/virtualenvs/pyleniumio-uteKMdtY/lib/python3.8/site-packages/_pytest/config/__init__.py:1193: 
PytestConfigWarning: could not load initial conftests: /Users/mateuszsiewniak/Projects/pyleniumio/conftest.py
  self.issue_config_time_warning(
```

**Comment by matteoosh** [comment]
Temporary solution in https://github.com/ElSnoMan/pyleniumio/pull/127

**Comment by ElSnoMan** [comment]
Fix using is_checked on checkboxes #127
